### PR TITLE
Upgrade Flow to lastest

### PR DESCRIPTION
**Pull request by josemigallas**
## Description
Upgrades Flow and Flow-typed to the latest version.

The new version of Flow includes new "ignore" comments among other things and allows to ignore specific erorrs (no more unsafe $FlowFixMe everywhere).

## Verification
Run `yarn lint`. No errors should pop up.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,8 +46,8 @@
     "eslint-plugin-react": "^7.12.4",
     "eslint-plugin-standard": "^4.0.0",
     "file-loader": "^1.1.6",
-    "flow-bin": "^0.100.0",
-    "flow-typed": "^2.5.2",
+    "flow-bin": "^0.144.0",
+    "flow-typed": "^3.2.1",
```
Comment (damianpm) [inline package.json:50]: `yarn flow:install`,  fails with error:
```
UNCAUGHT ERROR: Error: Invalid npm libdef version! It appears to be a non-continugous range.
    at pkgVersionMatch (/opt/app-root/src/project/node_modules/flow-typed/dist/lib/npm/npmLibDefs.js:256:11)
    at defs.filter.def (/opt/app-root/src/project/node_modules/flow-typed/dist/lib/npm/npmLibDefs.js:281:82)
    at Array.filter (<anonymous>)
    at filterLibDefs (/opt/app-root/src/project/node_modules/flow-typed/dist/lib/npm/npmLibDefs.js:275:15)
    at findNpmLibDef (/opt/app-root/src/project/node_modules/flow-typed/dist/lib/npm/npmLibDefs.js:329:27)
error Command failed with exit code 1.
```

Updating to 3.3.1 fix it.
```suggestion
    "flow-typed": "^3.3.1",
```

**Review by damianpm — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (josemigallas) [reply package.json]: Yes, it's pointed out in the description

**Review by hallelujah — APPROVED** [review]

**Review by damianpm — APPROVED** [review]

**Review by didierofrivia — CHANGES_REQUESTED** [review]
I'm getting errors, pasting only the last one :



```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ spec/javascripts/Plans/components/DefaultPlanSelectCard.spec.jsx:69:39

Cannot call wrapper.find(...).invoke because property invoke is missing in ReactWrapper [1]. [prop-missing]

     spec/javascripts/Plans/components/DefaultPlanSelectCard.spec.jsx
     66│   const wrapper = mount(<DefaultPlanSelectCard {...props} />)
     67│
     68│   await act(async () => {
     69│     wrapper.find('DefaultPlanSelect').invoke('onSelectPlan')(plan)
     70│   })
     71│
     72│   expect(errorSpy).toHaveBeenCalledWith('An error ocurred. Please try again later.')

     flow-typed/npm/enzyme_v3.x.x.js
 [1] 18│     find(selector: UntypedSelector): this,



Found 29 errors
error Command failed with exit code 2.
```

**Comment by damianpm** [comment]
@didierofrivia did you run `yarn flow:install` ?

**Review by didierofrivia — APPROVED** [review]
Ok, after `flow:install` task I'm only getting an error related to lib moment.js... so def it's my setup.

**Pull request merged**